doc/Customization-guide/How-to-contribute/Documentation-contribution-and-style-guide/index.md

```diff
@@ -11,62 +11,168 @@ sort_idx: 30
+* Provide descriptions that do not require pictures, or provide both pictures and written descriptions.
+To test whether the writing is effective, try removing, one at a time, first the words and then the pictures.
+With only one method, can you still figure out what to do?
+* Use numbered lists for step-by-step procedures and bulleted list for an unordered set of items.
+* Use a bulleted list or headings to emphasize important points.
+* Minimize the number of steps in a procedure.
+Individuals who have cognitive impairments may have difficulty following procedures that have many steps.
+Keep the steps simple, and keep the user oriented.
+* Provide links to related content within the documentation to help readers get more information.
+
+## Style and tone
+* Use _please_ and _thank you_ judiciously.
+Avoid using _please_ in instructions unless some step in the procedure causes inconvenience for the user or represents a workaround for some system limitation.
+* Use _sorry_ only in error messages that result in serious problems for the user.
+Don’t use _sorry_ if the problem occurred during the normal functioning of the program, such as when the user needs to wait for a network connection to be found.
+* Don’t try to be funny.
+Jokes, slang, and sarcasm are context-specific and hard to translate and localize.
+What’s funny to you might offend or alienate some portion of your audience, so it’s best to avoid these rhetorical ­approaches.
+* Avoid overusing subjective terms such as _easy_, _fast_, _fun_, and so on.
+Instead, demonstrate that something is easy or fun by providing relevant information.
+* Use everyday words when you can, and avoid formal language that you wouldn’t use when speaking to someone in person.
 * Avoid inventing new words or assigning words an unusual meaning.
-* Avoid using please in instructions unless some step in the procedure causes inconvenience for the user or represents a workaround for some system limitation.
-
-## Headings and capitalization
+* Use short, plain words as much as possible.
+* Don’t make generalizations about people, countries, regions, and cultures, especially if the generalizations could be considered derogatory.
+* Avoid using culturally sensitive terms.
+* Omit needless words.
+Don’t use two or three words when one will do.
+* Use questions sparingly. While questions may support a friendlier tone, overuse of questions doesn’t support a trustworthy voice.
+Questions can work well when users actually do have them, not when we invent them on their behalf.
+* Use exclamation points sparingly.
+The overuse of exclamation points can make content seem effusive or ingratiating.
+Save exclamation points for when they count.
+* Be mindful of globalization and cultural considerations before you use colloquialisms or idioms.
+* Do not use non-English words or phrases, such as _de facto_ or _ad hoc_, in English content, even if you think these terms are generally known and understood.
```

Comment:
I propose that we have only two exceptions for this rule: etc. and e.g. They are used very often and their English equivalents are too long. I'll put this in my next commit.